docs/env.mdx

```diff
@@ -0,0 +1,33 @@
+.env files can be used to configure properties such as runtime ids, api keys, etc.
+
+> Note: If you're looking for more secure ways to store values you might consider the [secrets manager](/docs/secrets) or in an [object store](/docs/storage).
+
+### What order are `.env` files loaded in?
+
+Nitric will automatically load your .env files in the following order.
+
+```bash
+nitric run
```

Comment:
we are updating documentation with `nitric start` instead of run.

Reply:
Thanks, I accepted your suggestion.